lib/run-in-terminal.coffee

```diff
@@ -233,6 +237,16 @@ class ArgumentsRequester
                 @save_and_hide()
                 start_terminal(@path or "", @line_edit_model.getText())
 
+switch require('os').platform()
+  when 'darwin'
+    defaultLaunchfile = """osascript -e 'tell application \"Terminal\"' -e 'activate' -e 'tell application \"Terminal\" to do script \"cd \\\"{working_directory}\\\" && {launcher} \\\"{file_path}\\\" \"' -e 'end tell'"""
+    defaultLaunchdir = """osascript -e 'tell application \"Terminal\"' -e 'activate' -e 'tell application \"Terminal\" to do script \"cd \\\"{working_directory}\\\" \"' -e 'end tell'"""
+  when 'win32'
+    defaultLaunchfile = 'start /D {working_directory} C:\Windows\System32\cmd.exe /u /k cd "{working_directory}" & {launcher} "{file_path}"'
```

Comment:
I think `&&` and `&` is unnecessary for windows and linux at all. Since terminals for them have "execution" flag.